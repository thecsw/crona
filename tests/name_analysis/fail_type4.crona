main : bool array[10] (ints: int array[10], bytes: byte array[10]) {
    i : int;
    i = 0;
    ret : bool array[10];
    while (i < 10) {
        ret[i] = ints[i] == bytes[i];
    }
    return ret;
} 