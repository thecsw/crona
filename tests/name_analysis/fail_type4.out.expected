main(int array [10],byte array [10]->bool array [10]):bool array[10](ints(int array [10]):int array[10], bytes(byte array [10]):byte array[10]){
	i(int):int;
	i(int) = 0;
	ret(bool array [10]):bool array[10];
	while ((i(int) < 10)){
		ret(bool array [10])[i(int)] = (ints(int array [10])[i(int)] == bytes(byte array [10])[i(int)]);
	}
	return ret(bool array [10]);
}